content/en/project/BiosignalEmotions/index.md

```diff
@@ -0,0 +1,206 @@
+---
+type: "project" # DON'T TOUCH THIS ! :)
+date: "2020-06-12" # Date you first upload your project.
+# Title of your project (we like creative title)
+title: "Biosignal processing for automatic emotion recognition"
+
+# List the names of the collaborators within the [ ]. If alone, simple put your name within []
+names: [Danielle Benesch, Achraf Kabbabi]
+
+# Your project GitHub repository URL
+github_repo: https://github.com/brainhack-school2020/Biosignal-Emotions-BHS-2020
+
+# If you are working on a project that has website, indicate the full url including "https://" below or leave it empty.
+website:
+
+# List +- 4 keywords that best describe your project within []. Note that the project summary also involves a number of key words. Those are listed on top of the [github repository](https://github.com/brainhack-school2020/project_template), click `manage topics`.
+# Please only lowercase letters
+tags: [classification, emotion, eeg, ecg]
+
+# Summarize your project in < ~75 words. This description will appear at the top of your page and on the list page with other projects..
+
+summary: "Can we automatically detect changes in emotions given a user's biosignals? In this project, we used multimodal biosignal data to predict the "target emotion" of audiovisual stimuli."
```

Comment:
the website build fails. I think it is because of the quotes around "target emotion". Could you try removing them?

Reply:
Yes, that was it. Thanks!